tokio/src/net/unix/ucred.rs

```diff
@@ -31,7 +31,7 @@ impl UCred {
     }
 }
 
-#[cfg(any(target_os = "linux", target_os = "android", target_os = "openbsd"))]
+#[cfg(any(target_os = "linux", target_os = "redox", target_os = "android", target_os = "openbsd"))]
 pub(crate) use self::impl_linux::get_peer_cred;
```

Comment:
Redox uses the same impl as Linux? That's easy then.

Reply:
The implementation of sockets and some details needs internal work in redox (then it will work the same). This change enables the build on redox so we can then work on the internal implementation.